apps/demos/testing/widgets/lookup/Lookup.test.js

```diff
@@ -0,0 +1,30 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+fixture('Lookup.Basics')
+  .page('http://localhost:8080/')
+  .beforeEach(async (t) => {
```

Comment:
This the same as here https://github.com/DevExpress/DevExtreme/pull/26891/files#r1521260396